### PR TITLE
Partially Fixes issue #55 and moved all fixture to conftest.py file.

### PR DESCRIPTION
**Pull request by jameerpathan111**

### CODE REVIEW

**Comment by ntkathole** [comment]
ok to test

**Comment by ntkathole** [comment]
[test]

**Comment by ntkathole** [comment]
ok to test

File: tests/conftest.py [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+    RHN_PASSWORD,
+    RHN_USERNAME,
+    RHN_POOLID,
+    upstream_url,
+)
+from testfm.log import logger
+
+
+@pytest.fixture(scope='function')
+def setup_for_hammer_defaults(request, ansible_module):
```
Comment (ntkathole) [inline tests/conftest.py]: @jameerpathan111 please add doc strings for each fixture. It will help others to understand which fixture is doing what.

Comment (jameerpathan111) [reply tests/conftest.py]: thanks, will add doc strings.

**Pull request merged**